uint32 seqID
uint32 RADARhwTimeStamp
float32[] rangeEstimate
float32[] velocityEstimate
float32[] amplitude

